### PR TITLE
Improved documentation for GDScript `in` keyword

### LINKED ISSUE #5824: "in" operator not well documented
**Your Godot version:** 3.4.4

**Issue description:** The "in" operator is described as a "content test" in the documentation. However, inexperienced coders (like me) don't know what this means (googling "content tests" was too vague) and it took me research to find it can be used to look for variable names in classes. You can see from this ( https://godotengine.org/qa/3099/check-if-script-of-a-node-has-variable ) that even people in the community seem to be unaware of it and don't use it. The GDScript basics documentation does not explain how it used nor do they give an example. In a dynamically typed language like GDScript it seems like operators like this would be really helpful. I think a paragraph needs to be added on the "in" operator and some examples of basic usage.

**URL to the documentation page (if already existing):** https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html

### PR DESCRIPTION
**Pull request by Spencer-Kotys**
This pull request resolves #5824 by improving the documentation of the 'in' keyword on the gdscript_basics page. An example of using the 'in' keyword was added to the if/else/elif section and more details were added to the operator description.

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -1046,6 +1051,16 @@ multiple lines to preserve readability::
             else "orange"
     print(fruit_alt)  # banana
 
+You may also wish to check if a value is contained within something. You can
+use an if statement combined with the 'in' operator to accomplish this::
```
Comment (Calinou) [inline tutorials/scripting/gdscript/gdscript_basics.rst:1055]:
```suggestion
You may also wish to check if a value is contained within something. You can
use an ``if`` statement combined with the ``in`` operator to accomplish this::
```

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 2]
```diff
@@ -264,7 +264,12 @@ The following is the list of supported operators and their precedence.
...
 | ``<`` ``>`` ``==`` ``!=`` ``>=`` ``<=``                                        | Comparisons                               |
 +--------------------------------------------------------------------------------+-------------------------------------------+
-| ``in``                                                                         | Content test                              |
+| ``in``                                                                         | When used with the 'if' keyword it checks |
+|                                                                                | if a value is within a string, list,      |
+|                                                                                | range, dictionary, or node.               |
+|                                                                                | When used with the 'for' keyword it is    |
+|                                                                                | used to iterate though the contents of a  |
+|                                                                                | string, list, range, dictionary or node.  |
```
Comment (Calinou) [inline tutorials/scripting/gdscript/gdscript_basics.rst:272]: Please amend this part to use the reStructuredText syntax for inline code blocks (surround "if" and "for" with **two** backticks, and do not put quotes).

Comment (Spencer-Kotys) [reply tutorials/scripting/gdscript/gdscript_basics.rst]: Syntax fix applied.

**Comment by akien-mga** [comment]
Cherry-picked for 3.6 and 3.5.

**Pull request merged**